lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -1052,13 +1152,33 @@ global proc usdMayaRef_changeGroupPrimNameText(string $primName)
 {
     global string $gUsdMayaReferenceUfePath;
 
+    // Validate the user input (note: the group prim name is allowed to empty in which case
+    // a default name will be created).
+    string $validatedGroupPrimName;
+    if ($primName != "") {
+        $validatedGroupPrimName = sanitizeName($primName);
+    }
+    if ($validatedGroupPrimName != $primName) {
+        textFieldGrp -edit -text $validatedGroupPrimName usdMayaReferenceGroupPrimName;
+    }
+
     // Fill-in the variant preview field.
     int $groupChecked = `checkBoxGrp -q -v1 usdMayaReferenceGroupGrp`;
     string $primPath = `python("import mayaUsdAddMayaReference; mayaUsdAddMayaReference.getPrimPath('" + $gUsdMayaReferenceUfePath + "')")`;
     if ($groupChecked) {
         string $variantPreviewPath = $primPath;
-        if ($primName != "") {
-            $variantPreviewPath += ($primPath + "/" + $primName);
+        if ($validatedGroupPrimName == "") {
+            string $ns;
+            string $sel = currentFileDialogSelection();
+            if ($sel == "") {
+                $ns = "<filename>";
+            } else {
+                string $filePath = fromNativePath($sel);
+                $ns = computeNamespace($filePath);
+            }
+            $variantPreviewPath += ("/" + $ns + "RNgroup");
+        } else {
+            $variantPreviewPath += ("/" + $validatedGroupPrimName);
```

Comment:
Fill-in field with either the group prim used (if non-empty) or a preview of what will be auto-generated when the name is left empty. Use a special tag <filename> if the user has not selected a file in the dialog yet.